### ISSUE TITLE
enhancement request: demo how to create multiple pages with fetchdata

### ISSUE THREAD

**Issue by anotherjesse** [sw-yx/ssg-demo#2]
1. thanks for this! I was starting to look at gatsbyjs but I prefer to stick with learning more about svelte

2. for folks new to ssg it might be useful to have the fetchpage demo show how to have different pages for each item in `data`  as well as the index.

Each item in data would be exposed at `/fetchdata/:slug`? 

And then update the index so it has an index to each page (expand `<pre>{JSON.stringify(data, null, 2)}</pre>` to be html with links?)

Hopefully this makes sense - if not pointers of how one should think about "data" in ssg (could just be a link to the relevant code in ssg)

**Comment by swyxio** [comment]
yeah no problem, i should really get on that but havent had the time recently. basically `ssg` for now really requires you to know `sapper` well, since it is truly meant to be a thin layer over sapper. so you create multiple pages the same way you do it in sapper. a `/src/routes/[id].svelte` file and sending a bunch of requests down that path from something that knows all the id's, e.g. `src/routes/index.svelte` links to `/foo` and /`bar`, so that the `[id].svelte` file receives a prop with `id` equal `foo` and `bar`. make sense?

**Comment by anotherjesse** [comment]
@sw-yx I haven't really dove into sapper yet - but I think your comment has enough information that I can research / learn what I need to make a PR (assuming this would be valuable to others as well)

**Comment by swyxio** [comment]
yup. go ahead and dive into sapper first, this thing is only a thin layer over sapper. for now. in the future i may want to leave sapper altogether, because i have collected enough pain points and explored this problem area enough that i have differing opinions.

for one thing.. i'm interested in [taking out clientside routing](https://twitter.com/ryanflorence/status/1186515553285857280) 😈  what do you think?

**Comment by anotherjesse** [comment]
Hrm, I like it for things like static sites.

It feels like it can simplify things a lot from browser / user perspective

But while I've been building static websites using various techniques for well over a decade, I haven't ever done it in JS and thought through what that could enable.  Enabling an intermixing of static and dynamic browsing experiences.  

That said, simple is better, and truly static is simple

**Comment by swyxio** [comment]
correct. the idea is not needing to change paradigms as you go from "truly static" to adding some dynamic features per-page.
